src/UI/templates/default/Glyph/tpl.glyph.standard.html

```diff
@@ -1,3 +1,3 @@
-<a class="glyph<!-- BEGIN highlighted --> highlighted<!-- END highlighted -->"<!-- BEGIN with_action --> href="{ACTION}"<!-- END with_action --> aria-label="{LABEL}"<!-- BEGIN with_id --> id="{ID}"<!-- END with_id -->>
+<a class="glyph<!-- BEGIN highlighted --> highlighted<!-- END highlighted --><!-- BEGIN disabled --> disabled<!-- END disabled -->"<!-- BEGIN with_action --> href="{ACTION}"<!-- END with_action --> aria-label="{LABEL}"<!-- BEGIN with_aria_disabled --> aria-disabled="{ARIA_DISABLED}"<!-- END with_aria_disabled --><!-- BEGIN with_id --> id="{ID}"<!-- END with_id -->>
```

Comment:
I think there is nothing like a disabled a-tag. We should use a span or omit the container tag completely instead imo.

Reply:
Ok, I saw the discussion in the other PR. We might keep it this way.